client/src/components/user/dashboard.js

```diff
@@ -1,39 +1,280 @@
+              <FontAwesomeIcon icon={faTimes} />
+            </button>
+          </div>
+          <UpdateSubscription
+          updateSubscription={this.handleUpdate}
+          showSubscriptionList={this.showSubscriptionList}
+          toggleLoadingState={this.toggleLoadingState}
+          prevSubscription={editingSubscription}
+          />
+        </div>;
     return (
-      <div>
-        <header>
-          <h1>Hello this is the users dashboard page.</h1>
-          <section className="subscription-list">
-            <SubscriptionsList />
-          </section>
-          {successMessage}
-          <a href={href}>
-            Sign Out!
-            <FontAwesomeIcon icon={faSignOutAlt} />
+      <div className="h-100 d-flex flex-column">
+        <nav className="navbar p-2 d-flex justify-content-around justify-content-md-between text-primary">
+          <a className="navbar-brand d-flex text-primary" href="#changeThis">
+            <img src={logo} alt="wateringCanIcon" height="40" />
+            <div className="d-none d-md-block">Water Your  Subs</div>
           </a>
-        </header>
+          <h1>{`${new Date().toDateString()}`}</h1>
+          <div className="d-flex dropdown">
+            <a className="dropdown-toggle" href="#dashboard" id="navbarDropdownMenuLink" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+              <img src={pfp} alt="user-pfp" height="40" />
+            </a>
+            <ul className="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdownMenuLink">
+              <li>
+                <a
+                  className="btn dropdown-item"
+                  data-bs-toggle="offcanvas"
+                  href="#offcanvasNotifications"
+                  role="button"
+                  aria-controls="offcanvasNotifications"
+                >
+                  Notifications
+                </a>
+              </li>
+              <li>
+                <a className="dropdown-item" href={href}>
+                  Sign Out
+                  <FontAwesomeIcon icon={faSignOutAlt} />
+                </a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+        <main className="d-flex flex-fill flex-column flex-md-row justify-content-between">
+          <div className="col-md-8 flex-fill h-100 d-flex align-items-center justify-content-center" id="mainContainer" >
+            {this.renderMainComponent(subscriptionForm)}
+          </div>
+          <div className="col-md-4 p-3 order-md-first flex-fill">
+            <SubscriptionsList
+              subscriptions={subscriptions}
+              setEditingSubscription={this.setEditingSubscription}
+              setActiveSubscription={this.setActiveSubscription}
+              handleDelete={this.handleDelete}
+            />
+            <div className="col">
```

Comment:
I would add a little margin above here so that the dotted line isn't squished up against the existing subscription list

```suggestion
            <div className="col mt-2">
```